### PR TITLE
Case Study Home Page

### PR DESCRIPTION
**Pull request by AndrewBestbier**
### Motivation
Hi Team!

Today, I present you with the Case Study part of the Home Page. The design links can be found here:
[Desktop View](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57e003a43cd939017f48e3ae)
[Mobile View](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57e29b01414fd0b82d1b6898)

You will notice that there is no design of the Tablet view, but there is an implementation of the Tablet in this code. This was done through conversations with Monika. 

gif
### Pre-flight checklist
- [x] Unit tests
- [x] Has been design reviewed
- [x] Add a gif - `gif`

### CODE REVIEW

File: site/components/case-study/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import InlineSVG from 'svg-inline-react';
+import styles from './style.css';
+
+/* SVG logo imports */
```
Comment (AndrewBestbier) [inline site/components/case-study/index.js]: Fix this comment to say PNG

File: site/components/case-study/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import skyFigureSVG from './SVG/sky-figure.svg';
+
+const CaseStudy = () => (
+  <div className={styles.caseStudyContainer}>
+    <h2 className={styles.heading}>We solve complex problems and deliver real impact.</h2>
+    <div className={styles.figuresContainer}>
+      <div>
+        <InlineSVG src={fortnumFigureSVG} className={styles.caseFigure} />
+        <span className={styles.caseText}>Number of awards for the new online store</span>
+        <img alt="Example Alt Here" src={fortnumPNG} className={styles.caseLogo} />
```
Comment (AndrewBestbier) [inline site/components/case-study/index.js]: Add correct Alt text

File: site/components/case-study/index.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <InlineSVG src={camdenFigureSVG} className={styles.caseFigure} />
+        <span className={styles.caseText}>Weeks to deliver a new online platform</span>
+        <img alt="Example Alt Here" src={camdenPNG} className={styles.caseLogo} />
+      </div>
+      <div>
+        <InlineSVG src={skyFigureSVG} className={styles.caseFigure} />
+        <span className={styles.caseText}>
+          Drop in customers pushing the 'need more help' button
+        </span>
+        <img alt="Example Alt Here" src={skyPNG} className={styles.caseLogo} />
```
Comment (AndrewBestbier) [inline site/components/case-study/index.js]: Do we actually use case Logo?

File: site/components/case-study/style.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
+@value black, white, linesOnBlack from "../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../css/_breakpoints.css";
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Space between imports and css

File: site/components/case-study/style.css [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+.heading {
+  composes: fontL from "../../css/typography/_fonts.css";
+  max-width: 1000px;
+  margin: 0px auto;
+  color: white;
+  text-align: center;
+  padding: 20px;
+}
+.buttonContainer {
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Move further down the page to represent the correct order. Remove flex-wrap

File: site/components/case-study/style.css [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 30px;
+}
+.caseStudyContainer {
+  background-color: black;
+  padding-bottom: 12px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: One line these paddings

File: site/components/case-study/style.css [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+  margin-bottom: 10px;
+}
+@media mediumScreen  {
+  .heading {
+    font-size: 40px;
+    padding: 40px;
+  }
+  .figuresContainer div {
+    width: 49%;
+    padding: 60px 0px 60px 0px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Could just be 60px 0px

File: site/components/case-study/style.css [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+  }
+  .figuresContainer .caseLogo {
+    height: 80px;
+  }
+  .caseCompanies img {
+    margin: 0px 15px;
+    height: 80px;
+  }
+  .buttonContainer {
+    padding: 60px 0px 70px 0px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: padding: 60px 0px;

File: site/components/case-study/index.js [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    <div className={styles.caseCompanies}>
+      <img alt="The logo of Selfridges" src={selfridgesPNG} />
+      <img alt="The logo of the Financial Times" src={financialTimesPNG} />
+      <img alt="The logo of the BBC" src={bbcPNG} />
+      <img alt="The logo of BMW" src={bmwPNG} />
+      <img alt="The logo of Lloyds Bank" src={lloydsPNG} />
+      <img alt="The logo of Car Trawler" src={cartrawlerPNG} />
+    </div>
+    <div className={styles.buttonContainer}>
+      <a className={styles.button}>See more of our work</a>
```
Comment (asavin) [inline site/components/case-study/index.js]: Should link to https://red-badger.com/our-work/

File: site/components/case-study/style.css [hunk 10]
```diff
@@ -0,0 +1,126 @@
+@value black, white, linesOnBlack from "../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../css/_breakpoints.css";
+
+.heading {
+  composes: fontL from "../../css/typography/_fonts.css";
+  max-width: 1000px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: variable plz

File: site/components/case-study/style.css [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+  margin: 0px auto;
+  color: white;
+  text-align: center;
+  padding: 20px;
+}
+.caseStudyContainer {
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: new line before each block :)

File: site/components/case-study/style.css [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+  background-color: black;
+  padding: 0px 5% 12px 5%;
+  composes: dividerBlack from "../../css/_divider.css";
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: composes should be at the top?

File: site/components/case-study/style.css [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+  composes: fontL from "../../css/typography/_fonts.css";
+  max-width: 1000px;
+  margin: 0px auto;
+  color: white;
+  text-align: center;
+  padding: 20px;
+}
+.caseStudyContainer {
+  background-color: black;
+  padding: 0px 5% 12px 5%;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: 5% should we a variable as well?

File: site/components/case-study/style.css [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+.figuresContainer img {
+  height: 50px;
+}
+.caseText {
+  color: white;
+  text-align: center;
+  font-size: 24px;
+  padding: 30px;
+  composes: fontXL from "../../css/typography/_fonts.css";
+  font-weight: bold;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: composes bold from file?

File: site/components/case-study/style.css [hunk 15]
```diff
@@ -0,0 +1,126 @@
...
+.figuresContainer .caseFigure svg {
+  height: 80px;
+}
+.figuresContainer img {
+  height: 50px;
+}
+.caseText {
+  color: white;
+  text-align: center;
+  font-size: 24px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: should come from typography

File: site/components/case-study/index.js [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+import tescoPNG from './PNG/tesco.png';
+
+/* SVG figure imports */
+import tescoFigureSVG from './SVG/tesco-figure.svg';
+import fortnumFigureSVG from './SVG/fortnum-figure.svg';
+import camdenFigureSVG from './SVG/camden-figure.svg';
+import skyFigureSVG from './SVG/sky-figure.svg';
+
+const CaseStudy = () => (
+  <div className={styles.caseStudyContainer}>
```
Comment (zoeabryant) [inline site/components/case-study/index.js]: please make a `section` not a div

File: site/components/case-study/style.css [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+  padding: 30px;
+  composes: fontXL from "../../css/typography/_fonts.css";
+  font-weight: bold;
+}
+.caseCompany {
+  height: 50px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Check if this impacts the > img select

File: site/components/case-study/style.css [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+  margin-bottom: 10px;
+}
+.buttonContainer {
+  display: flex;
+  justify-content: center;
+  padding: 30px;
+}
+.button {
+  composes: primaryButton from "../../css/_links.css";
+  font-size: 24px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: should come from typography

File: site/components/case-study/style.css [hunk 19]
```diff
@@ -0,0 +1,126 @@
...
+  padding: 15px 31px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: 15px 30px

File: site/components/case-study/style.css [hunk 20]
```diff
@@ -0,0 +1,126 @@
...
+@media mediumScreen  {
+  .heading {
+    font-size: 40px;
+    padding: 40px;
+  }
+  .figuresContainer div {
+    width: 49%;
+    padding: 60px 0px;
+  }
+  .figuresContainer div:nth-child(odd){
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: shpace

File: site/components/case-study/style.css [hunk 21]
```diff
@@ -0,0 +1,126 @@
...
+  .caseCompanies img:first-child {
+    margin-left: 0px;
+  }
+  .caseCompanies img:last-child {
+    margin-right: 0px;
+  }
+}
+@media largeScreen {
+  .heading {
+    font-size: 60px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: typography bro

File: site/components/case-study/style.css [hunk 22]
```diff
@@ -0,0 +1,126 @@
...
+    padding-top: 50px;
+  }
+  .figuresContainer .caseFigure svg {
+    height: 150px;
+  }
+  .figuresContainer img {
+    height: 80px;
+  }
+  .caseText {
+    font-size: 40px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: typo... graphy?

File: site/components/case-study/style.css [hunk 23]
```diff
@@ -0,0 +1,126 @@
...
+  }
+  .caseCompanies img:last-child {
+    margin-right: 0px;
+  }
+}
+@media largeScreen {
+  .heading {
+    font-size: 60px;
+    padding: 60px;
+    padding-top: 50px;
```
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: join these two lines

**Comment by AndrewBestbier** [comment]
Beautiful! I've done such a great job <3

File: site/components/case-study/index.js [hunk 24]
```diff
@@ -0,0 +1,66 @@
...
+    <h2 className={styles.heading}>We solve complex problems and deliver real impact.</h2>
+    <div className={styles.figuresContainer}>
+      <div>
+        <InlineSVG src={fortnumFigureSVG} className={styles.caseFigure} />
+        <span className={styles.caseText}>Number of awards for the new online store</span>
+        <img alt="The logo of Fortnum & Masons" src={fortnumPNG} />
+      </div>
+      <div>
+        <InlineSVG src={camdenFigureSVG} className={styles.caseFigure} />
```
Comment (zoeabryant) [inline site/components/case-study/index.js]: I'm concerned about the accessibility loss of using SVGs for these numbers - do we have alt text? Can we just use real text and icons when needed?

File: site/components/case-study/index.js [hunk 25]
```diff
@@ -0,0 +1,66 @@
...
+        <span className={styles.caseText}>Weeks to deliver a new online platform</span>
+        <img alt="The logo of Camden Market" src={camdenPNG} />
+      </div>
```
Comment (zoeabryant) [inline site/components/case-study/index.js]: can we use repeatable react components for this structure?

**Comment by lithin** [comment]
Beautiful!

File: site/components/case-study/style.css (hunk 10 above)
Comment (zoeabryant) [inline site/components/case-study/style.css]: please discuss with a designer what the max width for all elements on the homepage should be - we are using `1000px` in some places and `1080px` in others

File: site/components/case-study/style.css (hunk 13 above)
Comment (zoeabryant) [inline site/components/case-study/style.css]: only if it gets reused - i think this is only really relevant to this section

File: site/components/case-study/style.css (hunk 18 above)
Comment (zoeabryant) [inline site/components/case-study/style.css]: should come from `primaryButton` itself

File: site/components/case-study/style.css [hunk 26]
```diff
@@ -0,0 +1,126 @@
...
+}
+.button {
+  composes: primaryButton from "../../css/_links.css";
+  font-size: 24px;
+  padding: 15px 31px;
+}
+
+@media mediumScreen  {
+  .heading {
+    font-size: 40px;
```
Comment (zoeabryant) [inline site/components/case-study/style.css]: should come from typography 😄

File: site/components/case-study/style.css (hunk 11 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 13 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Not done as per discussion

File: site/components/case-study/style.css (hunk 12 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 14 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 18 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 19 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 20 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 21 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/style.css (hunk 22 above)
Comment (AndrewBestbier) [inline site/components/case-study/style.css]: Done

File: site/components/case-study/index.js (hunk 16 above)
Comment (AndrewBestbier) [inline site/components/case-study/index.js]: Done

File: site/components/case-study/index.js (hunk 25 above)
Comment (AndrewBestbier) [inline site/components/case-study/index.js]: Not done as per discussion

**Pull request merged**